README.md

```diff
@@ -123,9 +129,14 @@ Create a new w3 space with an optional name.
 
 List spaces known to the agent.
 
-### `w3 space register <email>`
+### `w3 space register [email]`
```

Comment:
we may want to reserve the positional arg here for `provider`, or dont support any positional optional args here, and just afford for `--provider` and/or `--email`.

rationale: I predict it might be more common to wanna do `w3 space register [provider]` than `w3 space register [email]`
Let's see what people try before deciding what the positional args are.

Reply:
nvm, I realized we've had that positional arg for awhile

Reply:
Hmm, perhaps move both to options?

Reply:
I like moving both to options yep